crates/bevy_render/src/camera/camera.rs

```diff
@@ -104,21 +104,45 @@ impl Default for Camera {
 }
 
 impl Camera {
+    /// Converts a physical size in this `Camera` to a logical size.
+    #[inline]
+    pub fn physical_to_logical(&self, input: UVec2) -> Option<Vec2> {
+        let scale = self.computed.target_info.as_ref()?.scale_factor;
+        Some((input.as_dvec2() / scale).as_vec2())
+    }
```

Comment:
I think you suggestion of naming this `to_logical` is good if we rename the input parameter as well.

```suggestion
    pub fn to_logical(&self, physical_size: UVec2) -> Option<Vec2> {
        let scale = self.computed.target_info.as_ref()?.scale_factor;
        Some((input.as_dvec2() / scale).as_vec2())
    }
```